packages/@ember/-internals/glimmer/tests/integration/application/debug-render-tree-test.ts

```diff
@@ -4,6 +4,8 @@ import {
   moduleFor,
   strip,
 } from 'internal-test-helpers';
+import { ExpectDeprecationFunc } from '../../../../../../internal-test-helpers/lib/ember-dev/deprecation';
```

Comment:
This should likely be added to the above imports from `internal-test-helpers`, and the `ExpectDeprecationFunc` type/interface should be exported from that package